src/wallet/rpcwallet.cpp

```diff
@@ -1268,6 +1289,7 @@ UniValue listreceivedbyaddress(const UniValue& params, bool fHelp)
             "    \"address\" : \"receivingaddress\",  (string) The receiving address\n"
             "    \"account\" : \"accountname\",       (string) DEPRECATED. The account of the receiving address. The default account is \"\".\n"
             "    \"amount\" : x.xxx,                  (numeric) The total amount in " + CURRENCY_UNIT + " received by the address\n"
+            "    \"amountZat\" : xxxx                 (numeric) The amount in zatoshis\n"
```

Comment:
Add `MINOR_CURRENCY_UNIT` set to "zatoshis", analogous to `CURRENCY_UNIT`. (The "Zat" suffixes on field names don't need to change.) This could be a separate PR if you prefer.